tools/graph-tool/pom.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.jboss.windup.web</groupId>
+        <artifactId>windup-web-parent</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+        <relativePath>../../</relativePath>
```

Comment:
It failed on CI without that line. That tells me that the line somehow worked, but you are also correct that the line is missing the "pom.xml" bit. I'm surprised that it worked. :)

I will fix it.